File: 1dam/Programacion/Colecciones y genéricos-20240521/EjemGenericos/EjemGenericos/src/Persona.java
public class Persona {
    int edad;
    String nombre;

    public Persona(int edad, String nombre) {
        this.edad = edad;
        this.nombre = nombre;
    }

    @Override
    public String toString() {
        return "Persona{" +
                "edad=" + edad +
                ", nombre='" + nombre + '\'' +
                '}';
    }
}

File: 2dam/SistemasGestionEmpresarial/ia_api/models/persona.py
from pydantic import BaseModel


class Persona(BaseModel):
    nombre: str = "Desconocido"
    apellido: str | None = None
    foto: str | None = None
    edad: int | None = None

class PersonaRead(Persona):
    id: int | None = 0

class PersonaCreate(Persona):
    pass

class PersonaUpdate(Persona):
    nombre: str|None